packages/edit-site/src/components/post-fields/index.js

```diff
@@ -242,6 +242,16 @@ function usePostFields( viewType ) {
 						label: name,
 					} ) ) || [],
 				render: PostAuthorField,
+				sort: ( a, b, direction ) => {
```

Comment:
If `sort` receives the item instead of the resolved value, we can calculate the author name based on the underlying data (`item._embedded?.author[ 0 ]?.name`).

It doesn't change anything in this particular case, though it means that if the data is part of the dataset, the Fields don't have to have any other dependency (in this case, `authors` would have to be added to the dependencies if it wasn't already).